bitnami/metallb/templates/speaker/daemonset.yaml

```
@@ -35,6 +35,10 @@ spec:
       serviceAccountName: {{ include "metallb.speakerServiceAccountName" . }}
       terminationGracePeriodSeconds: {{ .Values.speaker.daemonset.terminationGracePeriodSeconds }}
       hostNetwork: true
+      {{- if gt (len .Values.speaker.initContainers) 0 }}
+      initContainers:
+      {{ toYaml .Values.speaker.initContainers | nindent 8 }}
```

And for consistency with the rest of the file, tplValues here:
```suggestion
      {{- include "common.tplvalues.render" (dict "value" .Values.speaker.initContainers "context" $) | nindent 8 }} 
```